README.md

```diff
@@ -13,12 +13,13 @@
 
 ## Introduction
 
-Great that you've made it to this little Eclipse project! Let's get you started by providing a quick background
+Great that you've made it to this neat Eclipse project! Let's get you started by providing a quick background
 tour, introducing the project scope and all you need for installation and a first running example.
 
 So first off: What is iceoryx?
 
-iceoryx is an inter-process-communication (IPC) middleware for various operating systems (currently we support Linux, MacOS, QNX and Windows 10).
+iceoryx is an inter-process-communication (IPC) middleware for various operating systems (currently we support Linux,
+macOS, QNX, FreeBSD and Windows 10).
```

Comment:
FreeBSD uuuh yeah :D 

But maybe we can state that we should support unix posix systems in general but run our CI only on FreeBSD as a representative.

Reply:
That one is explicitly mentioned further down in the `Supported Platforms` section